app/js/app.js

```diff
@@ -154,34 +154,7 @@ ddsApp.config(function($locationProvider, $stateProvider, $urlRouterProvider, $u
             templateUrl: '/partials/foyer/patrimoine.html',
             controller: 'FoyerPatrimoineCtrl'
         })
-        .state('situation', {
-            url: '/situations/:situationId',
-            template: '',
-            controller: function(SituationService, $state, $stateParams) {
-                SituationService.restoreRemote($stateParams.situationId).then(function() {
-                    $state.go('foyer.resultat', { situationId: $stateParams.situationId });
-                });
-            }
-        })
-        .state('infos_complementaires', {
-            url: '/infos-complementaires?droit',
-            data: {
-                pageTitle: 'Informations complémentaires - '
-            },
-            views: {
-                '@': {
-                    templateUrl: '/partials/form-infos-complementaires/layout.html',
-                    controller: 'FormInfosComplementairesCtrl'
-                },
-                'individus@infos_complementaires': {
-                    templateUrl: '/partials/form-infos-complementaires/individus.html'
-                },
-                'adresse@infos_complementaires': {
-                    templateUrl: '/partials/form-infos-complementaires/adresse-contact.html'
-                }
-            }
-        })
-        .state('download_cerfa', {
+        .state('foyer.download_cerfa', {
```

Comment:
So that the `download_cerfa` page feels more integrated to the user journey.
![image](https://cloud.githubusercontent.com/assets/11834997/13950885/e0eb96ec-f02d-11e5-97d1-37507352826a.png)